code/game/machinery/computer/communications.dm

```diff
@@ -420,10 +420,13 @@ var/const/CALL_SHUTTLE_REASON_LENGTH = 12
 		if(STATE_ALERT_LEVEL)
 			dat += "Current alert level: [get_security_level()]<BR>"
 			if(security_level == SEC_LEVEL_DELTA)
-				dat += "<font color='red'><b>The self-destruct mechanism is active. Find a way to deactivate the mechanism to lower the alert level or evacuate.</b></font>"
+				dat += "<A HREF='?src=\ref[src];operation=securitylevel;newalertlevel=[SEC_LEVEL_RED]'>Red</A>"

```

Comment:
No! This breaks two gamemodes, malf & nook ops. 

`The self-destruct mechanism is active. Find a way to deactivate the mechanism to lower the alert level or evacuate.`

besides that, `goodcurity` is not used, since we're not using the achievements.